### ISSUE TITLE
Create an appointment confirmation page

### ISSUE THREAD

**Issue by rothlisbergerc** [rothlisbergerc/GroomingSalonWebsite#19]
**issue #18 should be finished before tackling this issue**

After clicking the 'Make Appointment' button on the scheduling page  the user should be directed to this confirmation page that displays the appointment information:

- [ ] A success confirmation message/heading: "Success! Please review your appointment information and we will see you " + Date/Time + " !"
## Appointment information:
  - [ ] User's name
  - [ ] User's phone number
  - [ ] Pet's name
  - [ ] Date and Time of appointment

**Comment by rothlisbergerc** [comment]
I need to rethink this one now that I'm getting into the code.

**Comment by rothlisbergerc** [comment]
Closing this issue since the behavior and appointment information cannot be completed until the database is linked to the appointment object information. Will create a new issue for that.
